frontend/src/PagesAdmin/RecruitmentPositionFormAdminPage/RecruitmentPositionFormAdminPage.tsx

```diff
@@ -62,15 +74,19 @@ export function RecruitmentPositionFormAdminPage() {
     is_funksjonaer_position: position?.is_funksjonaer_position || false,
 
     tags: position?.tags,
+    interviewers: position?.interviewers || [],
     // TODO: Add necessary fields to form.
     // gang: gangId,
     // recruitment: recruitmentId,
-    // interviewers: position?.interviewers,
   };
 
+  const initialInterviewers =
+    position.interviewers && position.interviewers?.length > 0
```

Comment:
Er ikke det redundant?
```suggestion
    position.interviewers?.length > 0
```

Reply:
Nei, hvis ikke sjekker du undefined > 0., det gir ikke helt mening, position.interviewrs?, altså spørsmålstegnet er strengt tatt redundant

Reply:
undefined > 0 blir bare false

Reply:
ts klager fortsatt på denne løsningen ass

Reply:
position?.interviewers?.length > 0

Reply:
Trenger ikke ternary engang.

Reply:
Det funker ikke for meg ass

Reply:
<img width="935" alt="Screenshot 2023-12-19 at 17 29 59" src="https://github.com/Samfundet/Samfundet4/assets/54811233/3e168f5d-2ac2-4fda-be66-ba86f6977301">

Reply:
Dropp ternary

Reply:
Done

Reply:
Ser ut til å være outdated, resolver